test/onnx/dynamo/test_dynamo_with_onnxruntime_backend.py

```diff
@@ -64,6 +64,33 @@ def _test_model_numerically(
                 for baseline_elem, result_elem in zip(baseline_result, result):
                     torch.testing.assert_close(baseline_elem, result_elem)
 
+    def _test_model_numerically_with_provided_backend_and_default_backend(
+        self,
+        model,
+        dynamo_backend,
+        example_args_collection,
+    ):
+        """Test the model numerically with two ONNXRuntime backends.
+
+        It's useful to test local backend and global backend triggered by
+        backend="onnxrt" at the same time.
+        """
+
+        # Test provided backend.
+        self._test_model_numerically(
+            model,
+            dynamo_backend,
+            example_args_collection,
+        )
+
+        # Test default backend.
+        torch._dynamo.reset()
+        self._test_model_numerically(
+            model,
+            "onnxrt",
+            example_args_collection,
+        )
```

Comment:
why not different tests which would call `torch._dynamo.reset()` as part of the `setUp` and `tearDown`?

Reply:
I will do
```
    @parameterized.expand(
        [
            (True,),
            (False,),
        ]
    )
    def test_elementwise_function_single_output(self, test_local_backend: bool):
```

and use `test_local_function` to decide which backend to use.